### DISCUSSION TITLE
General Crowdin translation questions

### DISCUSSION THREAD

**Discussion by steve8x8** [NightscoutFoundation/xDrip#2339] [Ideas]
Hi all -

While fixing some typos in, and scrolling through, the German translations, I came up with a few questions/ideas:

1. How to address the user in languages which have different ways for "polite formal" and "colloquial" communication? Will it be "Sie" or "Du" or even "du" (non-capitalised)? Is there a general agreement already present?
2. What about compound words, like "Transmitter-Status"? Hyphens, in German, are generally preferred over white space, the rules are a bit different from English ones though. (Fixing this will affect a lot of established translations :( )
3. Apparently over time a few strings have undergone a major overhaul, as the English text no longer matches the internal name of the string. Such changes don't seem to be automatically forwarded to all languages (i.e., their translations marked as possibly obsolete). Examples: anything with `_sdcard_`; double (now optional) calibration; also some of the Graph colour stuff.

Is there a "style document" somewhere? Pointers welcome... thanks!

**Comment by Navid200** [comment]
There is a forum inside Crowdin in the xDrip project as well.
Have you posted these questions there as well?  
It may be a good idea.

**Reply by steve8x8** [reply]
I eventually found the "Discussions" over there - I had always entered via the file tree before. Thanks for the heads up!
